float32[] weight
---
bool success
string message
